src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -8180,4 +8180,10 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_AmbigOperator" xml:space="preserve">
     <value>Operator resolution is ambiguous between the following members: '{0}' and '{1}'</value>
   </data>
+  <data name="IDS_FeatureClosedClass" xml:space="preserve">
+    <value>closed classes</value>
```

Comment:
if the feature string is 'closed classes', can this be IDS_FeatureClosedClasses?  (same for a lot of the naming elsewhere).